be/src/storage/lake/lake_primary_index.h

```diff
@@ -72,6 +72,12 @@ class LakePrimaryIndex : public PrimaryIndex {
 
     void set_local_pk_index_write_amp_score(double score);
 
+    // |key_col| contains the *encoded* primary keys to be deleted from this index.
+    // The position of deleted keys will be appended into |new_deletes|.
+    //
+    // |rowset_id| The rowset that keys belong to. Used for setup rebuild point (cloud native index only).
+    Status erase(const TabletMetadataPtr& metadata, const Column& pks, DeletesMap* deletes, uint32_t rowset_id);
```

Comment:
Only the meaning parameters are commented, please comment on what this function does

Reply:
Done